src/content/docs/security/capabilities.mdx

```diff
@@ -195,11 +195,11 @@ to higher privileged windows.
 
 ## Schema Files
 
-Tauri generates a JSON schema with all the permissions that are available to your
-application so you have autocompletion in your IDE.
-To use the schema, set the `$schema` property to one of the schemas
-inside the `gen/schemas` directory, which are platform-specific.
-Usually you will set it to `../gen/schemas/desktop-schema.json` or
+Tauri generates JSON schemas with all the permissions available to
+each platform-specific application, allowing autocompletion in your IDE.
+To use a schema, set the `$schema` property in your configuration to one of the
+platform-specific schemas located in the `gen/schemas` directory. Usually
+you'll set it to `../gen/schemas/desktop-schema.json` or
```

Comment:
```suggestion
you will set it to `../gen/schemas/desktop-schema.json` or
```
I'm trying to reduce short forms like these in the docs because i was told a few times by non-english speakers that writing it out makes it a bit easier for them and/or their translators.